ci/integration/deploy-test.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: concourse.k8s.io/v1beta1
+kind: Pipeline
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: deploy
+spec:
+  exposed: true
+  pipelineString: |
+
+    github_source: &github_source
```

Comment:
If you add this to the existing file then you don't have to define things like `&github_source` or `&task_toolbox` as they're already defined. I suggest moving this job to the existing pipeline file rather than having it in a separate file.